src/parser/index.ts

```diff
@@ -288,7 +288,12 @@ namespace OtherExprRhs {
         seqConst('NOT IN', reservedWord('NOT'), reservedWord('IN'))
       )
     ),
-    parenthesized(lazy(() => select))
+    parenthesized(
+      oneOf<Select | Expression[]>(
+        lazy(() => select),
+        sepBy1(symbol(','), constantExpr)
```

Comment:
This should use `expression` instead of `constantExpr`, because e.g. the following is valid SQL:

```
select 1 in ((select 1), 99 - 98);
```

Here, `(select 1)` and `99 - 98` are exprissions but not constants (as matched by `constExpr`).